src/main/java/net/neoforged/neoforge/client/event/RegisterWoodTypeEvent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.event;
+
+import java.util.Map;
+import net.minecraft.client.renderer.Sheets;
+import net.minecraft.client.resources.model.Material;
+import net.minecraft.world.level.block.state.properties.WoodType;
+import net.neoforged.bus.api.Event;
+import net.neoforged.fml.event.IModBusEvent;
+import org.jetbrains.annotations.ApiStatus;
+
+public class RegisterWoodTypeEvent extends Event implements IModBusEvent {
```

Comment:
Event name should be `RegisterWoodTypesEvent`.

Reply:
done!